### PR TITLE
det-1869: add crowdstrike_expected to update_test sdk / cli

### PR DESCRIPTION
**Pull request by makk94**

### CODE REVIEW

File: python/sdk/prelude_sdk/controllers/build_controller.py [hunk 1]
```diff
@@ -30,9 +30,11 @@ def create_test(self, name, unit, technique=None, test_id=None):
         raise Exception(res.text)
 
     @verify_credentials
-    def update_test(self, test_id, name=None, unit=None, technique=None):
+    def update_test(self, test_id, name=None, unit=None, technique=None, crowdstrike_expected_outcome: EDRResponse = None):
         """ Update a test """
         body = dict()
+        if crowdstrike_expected_outcome:
+            body[['expected']] = dict(crowdstrike=crowdstrike_expected_outcome.value)
```
Comment (keitharobertson) [inline python/sdk/prelude_sdk/controllers/build_controller.py:37]: `[[`? There some cool Python I don't know here?

File: python/sdk/tests/test_build.py [hunk 2]
```diff
@@ -111,8 +111,10 @@ def test_list_tests(self, unwrap):
 
     def test_update_test(self, unwrap):
         updated_name = 'updated_test'
-        res = unwrap(self.build.update_test)(self.build, test_id=pytest.test_id, name=updated_name, technique='T1234.001')
+        res = unwrap(self.build.update_test)(
+            self.build,test_id=pytest.test_id, name=updated_name, technique='T1234.001', crowdstrike_expected_outcome=EDRResponse.PREVENT)
```
Comment (keitharobertson) [inline python/sdk/tests/test_build.py:115]: that doesn't look very alphabetical! Or do we not do that for arg lists? Also missing space between build,test_id

**Comment by makk94** [comment]
i should just close this PR and start over 😞

File: python/sdk/tests/test_build.py (hunk 2 above)
Comment (makk94) [reply python/sdk/tests/test_build.py]: shame on me. fixed

File: python/sdk/prelude_sdk/controllers/build_controller.py (hunk 1 above)
Comment (makk94) [reply python/sdk/prelude_sdk/controllers/build_controller.py]: if so, i dont know it either ha

**Comment by keitharobertson** [comment]
> i should just close this PR and start over 😞

Mahina missing an opportunity to alphabetize??? Are you feeling ok? 😆

File: python/sdk/tests/test_build.py [hunk 3]
```diff
@@ -112,7 +112,7 @@ def test_list_tests(self, unwrap):
     def test_update_test(self, unwrap):
         updated_name = 'updated_test'
         res = unwrap(self.build.update_test)(
-            self.build,test_id=pytest.test_id, name=updated_name, technique='T1234.001', crowdstrike_expected_outcome=EDRResponse.PREVENT)
+            self.build, test_id=pytest.test_id, crowdstrike_expected_outcome=EDRResponse.PREVENT, name=updated_name, technique='T1234.001')
```
Comment (keitharobertson) [inline python/sdk/tests/test_build.py:115]: I'm getting concerned about you!
```suggestion
            self.build, crowdstrike_expected_outcome=EDRResponse.PREVENT, name=updated_name, technique='T1234.001', test_id=pytest.test_id)
```

Comment (keitharobertson) [reply python/sdk/tests/test_build.py]: ahhh ok that makes sense. I'm cool either way you prefer it

**Review by keitharobertson — APPROVED** [review]

**Pull request merged**